props/saber_fett263_buttons.h

```diff
@@ -5784,7 +5890,11 @@ SaberFett263Buttons() : PropBase() {}
 #ifndef DISABLE_COLOR_CHANGE
         if (EndColorZoom()) return true;
         if (SaberBase::GetColorChangeMode() != SaberBase::COLOR_CHANGE_MODE_NONE) {
+#ifdef MENU_SPEC_TEMPLATE
```

Comment:
Since this is done in many places, maybe have a helper function for it.
Or maybe just do:

```
#ifndef MENU_SPEC_TEMPLATE
void ToggleColorChangeMode() override {
  OriginalColorChangeMode();
}
#endif
```